tests/ut/jit/test_dump_tag_outline.py

```
@@ -0,0 +1,194 @@
+
+def _dispatch_dump_vars(program: ir.Program) -> dict[str, list[str]]:
+    """Map each synthesised kernel-dispatch callee name to its sorted dump_vars
+    name_hints. Dispatches are the cross-function Calls to the outlined incore
+    functions (``*_incore_*``)."""
+    out: dict[str, list[str]] = {}
+
+    class _Collector(ir.IRVisitor):
+        def visit_call(self, op):
+            try:
+                name = op.op.name
+            except Exception:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)